src/main/fc/rc_controls.c

```diff
@@ -180,9 +191,21 @@ static void updateRcStickPositions(void)
     tmp |= ((rxGetChannelValue(YAW) > rxConfig()->mincheck) ? 0x02 : 0x00) << (YAW * 2);
     tmp |= ((rxGetChannelValue(YAW) < rxConfig()->maxcheck) ? 0x01 : 0x00) << (YAW * 2);
 
+#if !defined(USE_VARIABLE_PITCH)
     tmp |= ((rxGetChannelValue(THROTTLE) > rxConfig()->mincheck) ? 0x02 : 0x00) << (THROTTLE * 2);
     tmp |= ((rxGetChannelValue(THROTTLE) < rxConfig()->maxcheck) ? 0x01 : 0x00) << (THROTTLE * 2);
-
+#else
+    // woga65: On helicopter, use COLLECTIVE RC-channel value because it is assigned
+    // to the throttle stick while THROTTLE is most likely assigned to a switch.
+    // Keep '<< (THROTTLE * 2)' to not overly complicate things.
+    if (STATE(HELICOPTER)) {
```

Comment:
For helis, I would suggest requiring throttle hold to be engaged as well, by checking throttle value is bellow minimum here.

We probably need to document this somewhere, as this also assumes a linear throttle curve, which would not necessarily be true with a classic (endangered species) normal mode setup, where zero stick would not map to 0 pitch.

Reply:
I am not sure whether a classic normal mode setup should be supported at all. Once you are used to pulling COLLECTIVE all the way down to get landed, it is hard to get rid of this habit and use throttle hold instead.

As it comes to fixed-pitch helicopters, those can easily be taken care of by the regular MULTIROTOR platform. 

> For helis, I would suggest requiring throttle hold to be engaged as well, by checking throttle value is bellow minimum here.

True!

Reply:
@mmosca 
Sorry, I don't quite get it.

`    tmp |= ((rxGetChannelValue(THROTTLE) > rxConfig()->mincheck) ? 0x02 : 0x00) << (THROTTLE * 2);`
`    tmp |= ((rxGetChannelValue(THROTTLE) < rxConfig()->maxcheck) ? 0x01 : 0x00) << (THROTTLE * 2);`

Assuming THROTTLE = 3, 0x01 shifted to the left by 6 is 64, 0x02 shifted by 6 is 128.
So if the throttle value is between mincheck and maxcheck, the bits 6 and 7 of tmp are set.

Should not be checked for a stick command? Whether the stick is below mincheck (128) or above maxcheck (64)?

Since I know that it works as intended, I am pretty sure, my mind is playing a trick on me.

Reply:
done